src/test/ui/suggestions/suggest-let-for-assignment.stderr

```diff
@@ -0,0 +1,26 @@
+error[E0425]: cannot find value `x` in this scope
+  --> $DIR/suggest-let-for-assignment.rs:4:5
+   |
+LL |     x = 1;
+   |     ^ not found in this scope
+   |
+help: maybe you meant to write an assignment here
+   |
+LL |     let x = 1;
+   |     ~~~~~
+
+error[E0425]: cannot find value `x` in this scope
+  --> $DIR/suggest-let-for-assignment.rs:5:23
+   |
+LL |     println!("x: {}", x);
+   |                       ^ not found in this scope
```

Comment:
Ideally, when suggesting `let x` we would not emit the other errors referencing `x`. (This reminds me of an old idea I had of emitting a single `E0425` for each name per scope.)

Reply:
Yes，that will remove some noise, I need some time to think about this, maybe we will do it in another PR.